cmd/fleetctl/flags.go

```diff
@@ -57,3 +57,17 @@ func stdoutFlag() cli.Flag {
 func getStdout(c *cli.Context) bool {
 	return c.Bool(stdoutFlagName)
 }
+
+func byHostIdentifier() cli.Flag {
+	return &cli.StringFlag{
+		Name:  "host-identifier",
```

Comment:
this is `--host` in the figma

Reply:
![image](https://github.com/fleetdm/fleet/assets/1501415/9c4b59e3-cc27-4743-9432-46db4ef71bcd)
Yup found there is still a callout in the figma for --type